### DISCUSSION TITLE
Changelog organized

### DISCUSSION THREAD

**Discussion by rekcodocker** [nextcloud/all-in-one#3447] [Ideas] [answered]
### Is your feature request related to a problem? Please describe. 
I find the changelog very interesting and read it to know what has changed.
It is hard to find what is relevant to me. 

### Describe the solution you'd like
Another open source package (Darktable) groups changes together based on what they do for developers or end-users.
This could be one way of grouping changes:
* Broken backwards compatibility + special attention for upgrades
* Big changes in AIO
* Bug fixes
* Smaller upgrades or features
* Performance upgrades
* Actual Nextcloud version
* library&package version upgrades/bumps
* translations added/improved

### Additional context
I have taken inspiration from this program for their very readable change-summary:
https://www.darktable.org/2023/06/darktable-4.4.0-released/

**Comment by szaimen** [comment]
I found this: https://github.com/orgs/community/discussions/50886
Let me check if this also works here

**Reply by szaimen** [reply, marked as answer]
Done with https://github.com/nextcloud/all-in-one/commit/5a4f027478a98e142a097292f9061a2fc3eb571d
